salt/orchestrate/aws/mitx_qa_elb.sls

```diff
@@ -43,8 +32,10 @@ create_elb_for_edx_{{ edx_type }}:
           enabled: True
           timeout: 300
     - cnames:
-        {% for domain in domains[edx_type] %}
-        - name: {{ domain }}
+        {% for domain_key, domain in purpose.domains.items()  %}
+        {% if (app_type == 'live' and domain_key in ['lms', 'gitreload'])
+           or app_type == 'draft' %}
+        - name: {{ domain }}.
```

Comment:
Yes, because that is how the domain is represented in the zone file. If the `.` isn't present then it won't work properly.